libs/gltfio/include/gltfio/Animator.h

```diff
@@ -56,6 +56,18 @@ class UTILS_PUBLIC Animator {
      */
     void updateBoneMatrices();
 
+    /**
+     * Updates the bone matrices of the specified instance using the state of this animatior.
+     * This is useful if you have another instance that has the same skeleton as the asset of this animator,
+     * and you wish to apply the same animation to those instances (e.g. clothing).
+     * 
+     * NOTE: In most cases, you only need to use the updateBoneMatrices() method. This method is necessary
+     * only when you need to synchronize animations across multiple instances with the same skeleton.
+     * 
+     * @param instance The instance to update.
+     */
+    void updateBoneMatricesForInstance(FilamentInstance* instance);
```

Comment:
I feel like maybe we can formalize `addInstance` below instead of adding this method?  

We can do some validation of the the animation associated with the "new", added instance vs the one that's used to build this animator.

Reply:
I thought so as well first. However, the "problem" I found with that is that we are again creating all those channels and data for the new instance.
With the solution I have right now we don't need to recreate any data (however I admit that my solution is maybe a bit fragile as it relies on the name of the entities to synchronise entity transforms).